waitress/pantry/viewsets.py

```diff
@@ -0,0 +1,97 @@
+from datetime import date
+
+import pytz
+from django.shortcuts import get_object_or_404
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status as status_code
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
```

Comment:
There are a couple of unused imports here.

You can run `make lint` in the root of this project to help you handle some of these.